### PR TITLE
Fix description for Local Storage

### PR DESCRIPTION
**Pull request by ABCI-FJSE**
ローカルストレージの節の説明を修正しました。

### CODE REVIEW

File: en/docs/04.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## 4.1. Home Area {#home-area}
 
-Home area is the disk area shared by each interactive and compute node, which is available to all ABCI users by default. The disk quota is limited to 200GiB.
+Home area is the disk area of the Lustre file system shared by each interactive and compute node, which is available to all ABCI users by default. The disk quota is limited to 200GiB.
```
Comment (stakizawa) [inline en/docs/04.md:5]: 1分目修正案
Home area is the disk area of the Lustre file system shared by interactive and compute nodes, and is available to all ABCI users by default.

File: en/docs/04.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ stripe_count:  4 stripe_size:   1048576 stripe_offset: 10
 
 ## 4.2. Group Area {#group-area}
 
-Group area is the disk area shared by each interactive and compute node.
+Group area is the disk area of the GPFS file system shared by each interactive and compute node.
```
Comment (stakizawa) [inline en/docs/04.md:81]: Group area is the disk area of the GPFS file system shared by interactive and compute nodes.

File: en/docs/04.md [hunk 3]
```diff
@@ -97,19 +97,43 @@ The follwing table shows how to utilize local storage by two types of node.
 
 ### 4.3.1. Using as a local storage {#using-as-a-local-storage}
 
-The NVMe storage can be used as a local storage for each compute node.
-In this case, user can access to local storage by using environment variables `SGE_LOCALDIR`.
-When using this method, you need not to specify any additional options.
-
+The local storage installed into each compute node and memory intensive node is available as Local area without specifying any additional options.
```
Comment (stakizawa) [inline en/docs/04.md:100]: You can use local storages attached to compute or memory intensive nodes on which your jobs are running as temporal Local area without specifying any options.

File: en/docs/04.md [hunk 4]
```diff
@@ -97,19 +97,43 @@ The follwing table shows how to utilize local storage by two types of node.
...
 Note that the amount of the local storage you can use is determined by "Resource type". For more detail on "Resource type", please refer to [Job Executing Resource](03.md#job-executing-resource).
+The local storage path is different for each job and user can access to local storage by using [environment variables](03.md#environment-variables) `SGE_LOCALDIR`.
```
Comment (stakizawa) [inline en/docs/04.md:102]: user -> you

他のマニュアルでも、説明文の主語は概ねyouになっているようですので、youに統一してください。

File: en/docs/04.md [hunk 5]
```diff
@@ -97,19 +97,43 @@ The follwing table shows how to utilize local storage by two types of node.
...
+```
 
 !!! warning
     The file stored under `$SGE_LOCALDIR` directory is removed when job finished. The required files need to be moved to Home area or Group area in job script using `cp` command.
 
 ### 4.3.2. Using as a BeeOND storage {#using-as-a-beeond-storage}
 
 The set of NVMe storages of job assigned compute nodes can be used as a
-distributed shared file system (BeeGFS) on demand.
+distributed shared file system BeeGFS On Demand(BeeOND) on demand.
```
Comment (stakizawa) [inline en/docs/04.md:136]: By using BeeGFS On Demand (BeeOND), you can aggregate local storages attached to compute nodes on which your job is running to use as a temporal distributed shared file system.

You can use local storages or memory intensive nodes on which your jobs are running as temporal Local area without specifying any options.

File: en/docs/04.md [hunk 6]
```diff
@@ -97,19 +97,43 @@ The follwing table shows how to utilize local storage by two types of node.
...
 When using on demand BeeGFS (BeeOND), you need to submit job with `-l USE_BEEOND=1` option.
```
Comment (stakizawa) [inline en/docs/04.md:137]: BeeONDの説明を前文でしているので、短縮。

When using on demand BeeGFS (BeeOND), -> To use BeeOND,

**Comment by YouYoshi** [comment]
いただいた[コメント](https://github.com/aistairc/abci-docs/pull/155#pullrequestreview-439629014)に対して修正を実施いたしました。

File: en/docs/04.md [hunk 7]
```diff
@@ -83,9 +83,9 @@ To use Group area, "Usage Manager" of the group needs to apply "Add group disk"
 
 ## 4.3. Local Storage {#local-storage}
 
-In ABCI System, 1.6 TB NVMe storage is installed into each compute node and 1.8 TB SATA 3.0 storage is installed into each memory intensive node. There are two ways to utilize this storage as follows:
+In ABCI System, 1.6 TB NVMe storage is installed into compute nodes and 1.8 TB SATA 3.0 storage is installed into memory intensive nodes. There are two ways to utilize this storage as follows:
```
Comment (stakizawa) [inline en/docs/04.md:86]: こちらはeachでお願いします。あとNVMe storage のまえに「a」

In ABCI System, a 1.6 TB NVMe storage is installed into each compute node and a 1.8 TB SATA 3.0 storage is installed into each memory intensive node. There are two ways to utilize these storages as follows:

**Comment by YouYoshi** [comment]
いただいた[コメント](https://github.com/aistairc/abci-docs/pull/155#pullrequestreview-440811535)に対して修正を実施いたしました。

**Pull request merged**